File: src/main/resources/static/js/service/stockinfoService.js
//服务层
app.service('stockinfoService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../stockinfo/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../stockinfo/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../stockinfo/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../stockinfo/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../stockinfo/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../stockinfo/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../stockinfo/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/com/zzti/fruits/service/StockInfoService.java
package com.zzti.fruits.service;
import java.util.List;

import com.zzti.fruits.entity.PageResult;
import com.zzti.fruits.pojo.Goods;
import com.zzti.fruits.pojo.StockInfo;
import com.zzti.fruits.pojogroup.GroupStockInfo;

import javax.servlet.http.HttpServletResponse;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface StockInfoService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<StockInfo> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(StockInfo stockinfo);
	
	
	/**
	 * 修改
	 */
	public void update(StockInfo stockinfo);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public StockInfo findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Goods goods, int pageNum, int pageSize);

    PageResult searchPoi(int page, int rows, String stockCount, String fid, String sid);

	void stockInfoReport(HttpServletResponse response,String stockCount,String fid,String sid);
}
